ml-agents-envs/mlagents/envs/subprocess_env_manager.py

```diff
@@ -119,18 +119,18 @@ def _send_response(cmd_name, payload):
             elif cmd.name == "close":
                 break
     except (KeyboardInterrupt, UnityCommunicationException):
-        print("UnityEnvironment worker: environment stopping.")
+        logger.info(f"UnityEnvironment worker {worker_id}: environment stopping.")
```

Comment:
Cleaned this up to use `logger` and made it and the debug's below more consistent.